packages/components/pages/f-registration/CHANGELOG.md

```diff
@@ -3,6 +3,13 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v3.11.0
+------------------------------
+*November 11, 2022*
```

Comment:
Also just noticed this is next week 😄

Reply:
good spot! My pc is showing US date 🤦‍♀️